stories/Forms.story.jsx

```diff
@@ -162,4 +72,70 @@ storiesOf("Forms", module)
       </PageLayout>
     ))
   )
-  .add("form with validation", withInfo()(() => <FormWithValidation />));
+  .add(
+    "form with validation",
+    withInfo()(() => (
+      <Validate rules={rules}>
+        {({
+          onSubmit,
+          onBlur,
+          onChange,
+          errors: { name, lastName, email, terms, newsletter }
+        }) => (
+          <Row position="center">
+            <Col col={9}>
+              <Form submit={onSubmit}>
+                <Input
+                  onBlur={onBlur}
```

Comment:
what happens if you don't pass in onBlur?

Reply:
wont validate onBlur